File: count-integers-with-even-digit-sum/count-integers-with-even-digit-sum.cpp
class Solution {
public:
    int countEven(int num) {
        int cnt =  0 ;
        for(int i = 2 ; i <=num ; i++){
            int k = i ;
            int sum = 0;
            while(k!=0){
             sum += k%10;
                k /= 10;
            }
            if(sum%2==0){
            cnt++;
        }
        }
        return cnt;
    }
};

File: count-integers-with-even-digit-sum/count-integers-with-even-digit-sum.py
class Solution:
    def countEven(self, num: int) -> int:
        cnt = 0 
        for i in range(2,num+1):
            k = i
            rem = 0
            while k!=0:
                rem += k%10
                k = k//10
            if rem%2==0:
                cnt += 1
        return cnt

